src/index.html

```diff
@@ -11,7 +11,119 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="label">
+          Surname:
+          <input type="text">
+        </label>
+        <label class="label">
+          Name:
+          <input type="text" maxlength="10" minlength="3">
```

Comment:
go through [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) once again, if there are more then 2 attributes, each of them should start on a new line
```suggestion
          <input
            type="text"
            maxlength="10"
            minlength="3"
          >
```